src/struct_info_internal.json

```diff
@@ -47,5 +47,14 @@
               "name"
             ]
         }
+    },
+    {
+        "file": "async_callback.h",
```

Comment:
Is this info generated automatically somehow and then checked in?

Reply:
No, this file is where we declare headers in C that the JS code wants to refer to. The automatic part is that the offsets of the fields in the struct are computed and made available to JS; this file just says what to "bind" for JS at all.